infrastructure/cloud/environments/prod/main.tf

```diff
@@ -256,30 +267,108 @@ module "ecs_api_td" {
 
 # Create Web ECS Service
 module "ecs_web_service" {
-  source         = "../../modules/ECS/Service"
-  environment    = var.environment
-  app_name       = var.app_name
-  name           = "web"
-  ecs_cluster_id = module.ecs_cluster.ecs_cluster.id
-  ecs_td_arn     = module.ecs_web_td.ecs_td_arn
-  tg_arn         = module.tg_web.tg_arn
-  sg_id          = data.aws_security_group.app_sg.id
-  subnet_ids     = module.subnets.web_subnets_ids
-  port           = module.ecs_web_td.port
+  source           = "../../modules/ECS/Service"
+  environment      = var.environment
+  app_name         = var.app_name
+  name             = "web"
+  ecs_cluster_id   = module.ecs_cluster.ecs_cluster.id
+  ecs_td_arn       = module.ecs_web_td.ecs_td_arn
+  tg_arn           = module.tg_web.tg_arn
+  sg_id            = data.aws_security_group.app_sg.id
+  subnet_ids       = module.subnets.web_subnets_ids
+  port             = module.ecs_web_td.port
+  ecs_cluster_name = module.ecs_cluster.ecs_cluster.name
+  max_capacity     = 3
+}
+
+# Create CloudWatch Alarms for Web ECS Service
+module "ecs_web_alarms" {
+  source       = "../../modules/Cloudwatch/Alarms"
+  environment  = var.environment
+  app_name     = var.app_name
+  namespace    = "AWS/ECS"
+  service_name = "web-ecs-service"
+  dimensions = {
+    ClusterName = module.ecs_cluster.ecs_cluster.name
+    ServiceName = module.ecs_web_service.service_name
+  }
+  alarm_configurations = [
+    {
+      name                = "cpu-high"
+      metric_name         = "CPUUtilization"
+      comparison_operator = "GreaterThanThreshold"
+      threshold           = 70
```

Comment:
Can these be parameterized? Then we can have different settings in each env if needed.